6404-ERROR-Game Crash at Wed Jan 12 19:07:57 2022 (Version 0.13.0a)
Traceback (most recent call last):
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\main.py", line 109, in <module>
    game_state = run(game_state)
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\main.py", line 88, in run
    new_game_state, info_getter, bgm_song, settings, ruleset = get_game_state(game_state, cwd)
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\main.py", line 65, in get_game_state
    return ugs(game_state, cwd)
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\engine\game_handler.py", line 172, in update_game_state
    return game_state, info_getter, song_playing, settings, ruleset
UnboundLocalError: local variable 'info_getter' referenced before assignment
Game State: quit,
P1 Blob: quirkless, is CPU: False
P2_Blob: quirkless, is CPU: False

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

7212-ERROR-Game Crash at Wed Jan 12 19:24:28 2022 (Version 0.13.0a)
Traceback (most recent call last):
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\main.py", line 109, in <module>
    game_state = run(game_state)
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\main.py", line 88, in run
    new_game_state, info_getter, bgm_song, settings, ruleset = get_game_state(game_state, cwd)
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\main.py", line 65, in get_game_state
    return ugs(game_state, cwd)
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\engine\game_handler.py", line 84, in update_game_state
    info_getter = engine.gameplay.handle_gameplay(p1_blob, p2_blob, ruleset, settings, p1_is_cpu, p2_is_cpu, timer)
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\engine\gameplay.py", line 118, in handle_gameplay
    p1_blob.move(pressed)
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\engine\blobs.py", line 1021, in move
    self.ability()
TypeError: ability() missing 1 required positional argument: 'return_inc_weight'
Game State: casual_match
P1 Blob: doctor, is CPU: False
P2_Blob: ice, is CPU: False

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

6852-ERROR-Game Crash at Wed Jan 12 23:23:58 2022 (Version 0.13.0a)
Traceback (most recent call last):
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\main.py", line 109, in <module>
    game_state = run(game_state)
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\main.py", line 88, in run
    new_game_state, info_getter, bgm_song, settings, ruleset = get_game_state(game_state, cwd)
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\main.py", line 65, in get_game_state
    return ugs(game_state, cwd)
  File "c:\Users\LastenKONE\Documents\GitHub\blob_ball\engine\game_handler.py", line 172, in update_game_state
    return game_state, info_getter, song_playing, settings, ruleset
UnboundLocalError: local variable 'info_getter' referenced before assignment
Game State: quit,
P1 Blob: wind, is CPU: False
P2_Blob: fire, is CPU: False

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

